CMakeLists.txt

```diff
@@ -72,9 +89,16 @@ target_include_directories(mpark_variant INTERFACE
 if (ANDROID)
     set(FLATBUFFERS_BUILD_FLATC OFF CACHE BOOL "FLATBUFFERS_BUILD_FLATC" FORCE)
 endif()
+
+if(IOS)
+    set(FLATBUFFERS_BUILD_FLATC OFF CACHE BOOL "FLATBUFFERS_BUILD_FLATC" FORCE)
+    set(FLATBUFFERS_BUILD_TESTS OFF CACHE BOOL "FLATBUFFERS_BUILD_TESTS" FORCE)
+endif()
+
 set(FLATBUFFERS_BUILD_TESTS OFF CACHE BOOL "FLATBUFFERS_BUILD_TESTS" FORCE)
 add_subdirectory(third_party/flatbuffers EXCLUDE_FROM_ALL)
 
+if(NOT IOS)
```

Comment:
It did not seems necessary to build it for iOS, since the command lines cannot be executed on command iOS. 
So I installed it via homebrew on macos and modified the cmake  flatbuffers/CMakeLists.txt to execute the one installed.

Reply:
It is supposed to build a host version (running on the build machine, which presumably will be a x64 mac) - so that we can use the same flatc everywhere.
Is that not working?

Reply:
xcodebuild did not handle very well multiple target sdks (osx, ios) in the same project, and refused to build it successfully. 
One workaround to be sure to have the same flatc everywhere could be, to build it first on its own, before building the iOS project. 
I can eventually check there is a better workaround.